maestro-ios-driver/src/main/kotlin/xcuitest/XCTestDriverClient.kt

```diff
@@ -233,7 +233,7 @@ class XCTestDriverClient(
             } else {
                 val message = "Failed request for XCTest server"
                 val responseBody = """
-                    { "message" : "$message" }
+                    { "exceptionMessage": "${connectException.localizedMessage}, "stackTrace": "${connectException.stackTraceToString()} }
```

Comment:
This wouldn't be that useful if I am not wrong. Reason because IOException is going to be thrown mostly in cases:

1. Trying to reach the XCTest server when it's not yet started
2. Server crashed
3. Test got canceled
In most cases, we'll have connected exceptions extending the IOException we are catching here. AFAICT, more useful would be looking at benchmarks that are affected by this and validating if either of these three cases are true. Let me know your thoughts!